File: core-tests/e2e-tests/spring-rest-h2-v2/src/test/java/org/evomaster/e2etests/spring/h2/SpringTestBase.java
package org.evomaster.e2etests.spring.h2;

import org.evomaster.client.java.controller.EmbeddedSutController;
import org.evomaster.e2etests.utils.RestTestBase;

public class SpringTestBase extends RestTestBase {

    protected static void initClass(EmbeddedSutController controller) throws Exception {

        RestTestBase.initClass(controller);
    }
}

File: e2e-tests/spring-graphql/src/test/kotlin/org/evomaster/e2etests/spring/graphql/SpringTestBase.kt
package org.evomaster.e2etests.spring.graphql

import org.evomaster.client.java.controller.EmbeddedSutController
import org.evomaster.e2etests.utils.GraphQLTestBase
import org.evomaster.e2etests.utils.RestTestBase
import org.evomaster.e2etests.utils.WsTestBase

/**
 * Created by arcuri82 on 03-Mar-20.
 */
abstract class SpringTestBase : GraphQLTestBase() {

    companion object {
        @JvmStatic
        @Suppress("ACCIDENTAL_OVERRIDE")
        //https://youtrack.jetbrains.com/issue/KT-12993
        fun initClass(controller: EmbeddedSutController) {
            WsTestBase.initClass(controller)
        }
    }
}